add-on/panels/sidebar.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<!-- This Source Code Form is subject to the terms of the Mozilla Public
+   - License, v. 2.0. If a copy of the MPL was not distributed with this
+   - file, You can obtain one at http://mozilla.org/MPL/2.0/.  -->
+<html>
+  <head>
+    <meta charset="utf-8">
```

Comment:
We either need to add in a CSP for this in nsDocument.cpp, or we need to fix [Bug 1225120](https://bugzilla.mozilla.org/show_bug.cgi?id=1225120) to switch to having the CSP in the meta.

I've got some fresh ideas for how we can fix that bug nicely, so I'll see if I can move that forward tomorrow. In the meantime, I suggest adding an `XXX akita: Add meta tag for CSP after bug 1225120 is fixed` comment.